### ISSUE TITLE
Error opening freeldr.ini

### ISSUE THREAD

**Issue by pe200012** [maharmstone/quibble#73]
![image](https://user-images.githubusercontent.com/13463539/188318424-ea93cfd3-bfa5-43a1-8e5d-e764b7ebd1e0.png)

My GRUB config:
```
#!/bin/sh
exec tail -n +3 $0
# This file provides an easy way to add custom menu entries.  Simply type the
# menu entries you want to add after this comment.  Be careful not to change
# the 'exec tail' line above.
menuentry 'Quibble' {
        insmod part_gpt
        insmod fat
        search --no-floppy --set=root --fs-uuid "A66B-FA4C"
        chainloader /efi/quibble/quibble.efi
}
```

freeldr.ini
```
[FREELOADER]
TimeOut=10
DefaultOS=Windows

[Operating Systems]
Windows="Windows"
Windows_Debug="Windows (Debug)"

[Windows]
SystemPath=btrfs(9593f4b8-0c4f-4a43-8056-61953a08e088)\Windows
Options=/SUBVOL=106

[Windows_Debug]
SystemPath=btrfs(9593f4b8-0c4f-4a43-8056-61953a08e088)\Windows
Options=/SUBVOL=106 /DEBUG
```

efi partition:
```
/boot/efi
└── EFI
    ├── GRUB
    │   └── grubx64.efi
    └── quibble
        ├── drivers
        │   └── btrfs.efi
        ├── font.ttf
        ├── freeldr.ini
        └── quibble.efi
```

btrfs subvol list
```
ID 256 gen 475277 top level 5 path @root
ID 257 gen 475277 top level 5 path @home
ID 258 gen 475256 top level 5 path @data
ID 259 gen 474121 top level 5 path @swap
ID 260 gen 456120 top level 256 path @root/var/lib/portables
ID 261 gen 456120 top level 256 path @root/var/lib/machines
ID 262 gen 475161 top level 5 path @windows
```

**Comment by GlitchR9506** [comment]
I tried to use Quibble in VMWARE and got the same error, It worked when I moved quibble to the root `\` instead of `\efi\boot`

**Comment by SzczurekYT** [comment]
I get this same error, when trying to boot (chainload) Quibble from Grub, no matter where I put Quibble.
My UEFI doesn't support adding an entry for efi files so I'm stuck with chainloading from grub.
@maharmstone Is there any way to debug this? Maybe you could add some more prints around somewhere here https://github.com/maharmstone/quibble/blob/7b9962b6001a8f05dac8243fda3cf73324f57d6f/src/menu.cpp#L407, that would for example show the path / directory / partition used? And If you do please drop a build here for testing, because c/c++ dependency management is kinda non existent, resulting in building a c/c++ project being something that exceeds my abilities.

**Comment by maharmstone** [comment]
What happens if you use GRUB to run the EFI shell, then use the shell to run Quibble?

**Comment by SzczurekYT** [comment]
> > file_open returned EFI_ACCESS_DENIED
>
> That's not good. This is an error that Quibble is feeding through from the firmware, rather than one it's generating itself. I suspect your firmware is too broken for Quibble to work.
>
> Maybe try updating it to the latest version, if you've not done this already.

I updated it like 2-3 days ago.
One thing that came to my mind is Secure Boot. I started searching and I found out that my firmware has a submenu called Windows OS Settings, and in it there is an option called "Bios UEFI/CSM Mode" which was set to CSM.
So I set it to UEFI, and then Secure Boot option setting appeared, so I disabled it.
So now instead of CSM we have UEFI mode with Secure Boot disabled.
Maybe this will change something. I don't know what CSM is, but as far as I understand CSM is something meant for older systems.
With Grub no difference, still EFI_NOT_FOUND, maybe it will change something with EFI shell option. I will let you know when I will have some time to test this today or tomorrow.
